data/datavalid.yml

```diff
@@ -391,8 +391,8 @@ schemas:
       - name: uid
       - name: counsel
       - name: charging_supervisor
-      - name: resolution
-      - name: action
+      - name: appeal_disposition
+      - name: action_appealed
```

Comment:
Please add schema for appeal hearing as well, and run datavalid afterward.

Reply:
Thanks. Please review.